app/controllers/admin/split_searches_controller.rb

```diff
@@ -0,0 +1,10 @@
+class Admin::SplitSearchesController < AuthenticatedAdminController
+  def create
+    split = Split.find_by(name: params[:name])
```

Comment:
So, just confirming that this is intentional:
- this doesn't filter down to active splits
- if you've filtered the index down to a specific app, the search will ignore that filter

I think these are very reasonable choices. I don't think there's any reason you should'nt be able to get back to a fully retired split, as long as it's very clear that it's retired in the UI.

Reply:
Yes, the goal was to be able to find retired splits.

And yes, it was meant to ignore the app.